### PR TITLE
3762 markers as monitors

### PR DESCRIPTION
**Pull request by gurhar1133**
markers are converted to monitors. Four cases handled; markers & monitors, neither markers nor monitors and cases with one but not the other.

Also fixed bug where cross section panel renders even if no instruments

### CODE REVIEW

File: sirepo/template/controls.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from __future__ import absolute_import, division, print_function
+from aenum import enum
```
Comment (moellep) [inline sirepo/template/controls.py:8]: Looks like an unused import

File: sirepo/template/controls.py [hunk 2]
```diff
@@ -165,17 +166,29 @@ def write_parameters(data, run_dir, is_parallel):
...
+            _id=LatticeUtil.max_id(data) + 1,
+            name='M1',
+            type='MONITOR',
+        )
+        data.models.elements.append(m)
+        assert len(data.models.beamlines) == 1, \
+            f'expecting 1 beamline={data.models.beamlines}'
+        data.models.beamlines[0]['items'].append(m._id)
+        return
+    for i, e in enumerate(data.models.elements):
```
Comment (moellep) [inline sirepo/template/controls.py:184]: I think you could do this part as the first step in the method and then you don't need to count markers, only monitors.

File: sirepo/template/controls.py [hunk 3]
```diff
@@ -165,17 +166,29 @@ def write_parameters(data, run_dir, is_parallel):
...
+        if e.type == 'MARKER':
+            m = PKDict(
```
Comment (moellep) [inline sirepo/template/controls.py:186]: You can remove the temporary variable m and assign directly to data.models.elements[i]

**Comment by gurhar1133** [comment]
Made changes

**Pull request merged**